src/web/site_admin.ml

```diff
@@ -1090,8 +1090,8 @@ let () =
             in
             se.se_public_creds_received <- true;
             let service = preapply ~service:election_draft uuid in
-            Pages_common.generic_page ~title:(s_ "Success") ~service
-              (s_ "Credentials have been generated and mailed! You should download private credentials (and store them securely), in case someone loses his/her credential.") () >>= Html.send
+            Pages_common.generic_page ~title:(s_ "Done") ~service
+              (s_ "Credentials generated and sent via e-mail. Download your private credentials (and store them securely), in case someone loses theirs.") () >>= Html.send
           )
```

Comment:
This is a misinterpretation: here, the private credentials are all the voters', and each individual voter can lose their credential.